tests/cbs_ut/cbs_ut.c

```diff
@@ -67,6 +67,7 @@ static ON_AMQP_MANAGEMENT_ERROR saved_on_amqp_management_error;
 static void* saved_on_amqp_management_error_context;
 static ON_AMQP_MANAGEMENT_EXECUTE_OPERATION_COMPLETE saved_on_execute_operation_complete;
 static void* saved_on_execute_operation_complete_context;
+static ASYNC_OPERATION_HANDLE test_my_amqp_management_execute_operation_async_result = (ASYNC_OPERATION_HANDLE)0x4308;
```

Comment:
this is a random number right?

Reply:
Yes. That's standard practice for unit tests in the old C SDK and submodules.